cloudify_agent/api/pm/base.py

```diff
@@ -107,9 +101,47 @@ class Daemon(object):
         the password for the broker connection
         defaults to 'guest'
 
-    ``manager_port``:
+    ``file_server_host``:
+
+        the IP or hostname of the file server (Required)
+
+    ``rest_host``:
```

Comment:
I exposed it in a separate branch (CFY-5135-ssl-on-file-server) since it's required for the file-server ssl support. the file_server_host is required in this branch already, because I eliminated manager-ip and broke its uses to broker-ip, rest-host and file-server-host instead.